test/include/problems/DummyExternalProblem.h

```diff
@@ -0,0 +1,34 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#ifndef DUMMYEXTERNALPROBLEM_H
+#define DUMMYEXTERNALPROBLEM_H
+
+#include "ExternalProblem.h"
+
+class DummyExternalProblem;
+
+template <>
+InputParameters validParams<DummyExternalProblem>();
+
+/**
+ * FEProblemBase derived class that will fail a prescribed timestep for testing
+ * timestepping algorithms
```

Comment:
This seems like a copy pasted comment. I do not see how `DummyExternalProblem` could fail